### ISSUE TITLE
Installation for windows developers using Anaconda needs to be updated

### ISSUE THREAD

**Issue by Garyfallidis** [nipy/dipy#1084]
Our installation documentation for people working with Anaconda who want to develop in Windows needs to be updated. More information here.

Just wanted to give you a quick feedback about the meeting we had with Jon to try and install DIPY for development work in Windows 10 (64bit) using an Anaconda installation.

What we found is that if you are using recent Anaconda in Windows you do not need to install mingw or libpython (with conda). All you need after or before installing Anaconda is to install one of the free versions (or not) of the Visual C++ compiler (e.g. community edition). After that then cython will find the correct compiler and build the shared objects with no problem.
So, in summary the steps are:
1. Install Anaconda for your windows version.
2. Install Visual C++ (or Studio) community edition (or other).
3. pip install nibabel
4. conda install vtk
5. cd dipy;
6. python setup.py develop (or python setup.py build_ext --inplace)

I hope this helps. We need to update our installation page! Cygwin/mingw is no more for anaconda users. We should also check what happens with Openmp when Visual C++ is used. 

Many thanks to Jon for reporting the problem.

**Comment by skoudoro** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#1084](https://trello.com/c/DO5GzMir/15-1084)

**Comment by skoudoro** [comment]
closed by #1361
